tests/qos/test_buffer_traditional.py

```diff
@@ -0,0 +1,222 @@
+import logging
```

Comment:
add pytest topology marker 'any' for this file since these can run on t0 or t1 topology

Reply:
Fixed

> @stephenxs , isn't this testcases only applicable to 201911? Then we also need to add a version check in the beginning and skip if there is a version mismatch

Hi @neethajohn as this is the branch dedicated for 201911 branch, do we still need to check it?